instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb

```diff
@@ -24,24 +32,39 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
           !defined?(::ActiveSupport::Notifications).nil? && gem_version >= MINIMUM_VERSION
         end
 
+        def subscribe
+          SUBSCRIPTIONS.each do |subscription_name|
+            OpenTelemetry.logger.debug("Subscribing to #{subscription_name} notifications with #{_tracer}")
+            OpenTelemetry::Instrumentation::ActiveSupport.subscribe(_tracer, subscription_name, default_attribute_transformer)
+          end
+        end
+
         private
 
+        def _tracer
+          self.class.instance.tracer
+        end
+
         def gem_version
           Gem::Version.new(::ActiveModel::Serializer::VERSION)
         end
 
-        def require_dependencies
-          require_relative 'event_handler'
+        def install_active_support_instrumenation
```

Comment:
You will want to pass the config object here otherwise it will never receive the options.

Reply:
NVM. I have no idea what in the world I am talking about.